app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -186,6 +186,10 @@ def destroy_vm(uuid)
       true
     end
 
+    def supports_vms_pagination?
+      true
```

Comment:
see GH-3175, which would need to go in first to be able to update rbovirt  (which is also required for GH-2350)